### ISSUE TITLE
undefined local variable or method 'machine_specs'

### ISSUE THREAD

**Issue by mrmarbury** [chef/chef-provisioning-fog#224]
Hi,

I get the above error when I execute `chef provision --no-policy -r ostest`

my provision recipe: (sensitive Data removed)

```
require 'chef/provisioning'

with_chef_server "http://localhost:7799",
  :client_name => Chef::Config[:node_name],
  :signing_key_filename => Chef::Config[:client_key]

machine 'foobar' do
  machine_options({
      :bootstrap_options => {
          :image_ref => 'stuff',
          :nics => [{ :net_id => 'stuff'}],
          :key_name => 'automation',
          :floating_ip_pool => 'stuff'
      },
      :ssh_username => 'test'
  })
  action :setup
end
```

my knife.rb: (sensitive data  removed)

```
current_dir = File.dirname(__FILE__)
log_level                :info
log_location             STDOUT
cookbook_path            ["#{current_dir}/../cookbooks"]

driver 'fog:OpenStack'
driver_options :compute_options => {
    :openstack_auth_url => 'https://foobar:5000/v3',
    :openstack_username => 'stuff',
    :openstack_api_key  => 'stuff',
    :openstack_tenant   => 'stuff'
}

chef_server_url 'http://localhost:7799'
node_name       'foobar'
local_mode true
chef_zero.enabled true
```
Chef Zero is running on the port and responding to requests

I hope it is just a pebkac

**Comment by mrmarbury** [comment]
no idea?

**Comment by tas50** [comment]
@mrmarbury Sorry for the late response. This was fixed in 0.26.1, which is in the latest chefDK 2.x and 3.x
